packages/themes/scss/basil-green.scss

```diff
@@ -0,0 +1,28 @@
+@import url("https://fonts.googleapis.com/css?family=Pacifico");
```

Comment:
move this to potluck's CSS? Pacifico is not used in the theme. Also, add `&display=swap` to the end of the URL

Reply:
Resolved!